paasta_tools/autoscaling/autoscaling_service_lib.py

```diff
@@ -355,6 +365,80 @@ def get_error_from_utilization(utilization, setpoint, current_instances):
         return 0.0
 
 
+def get_autoscaling_info(marathon_client, service, instance, cluster, soa_dir):
+    service_config = load_marathon_service_config(
+        service=service,
+        instance=instance,
+        cluster=cluster,
+        soa_dir=soa_dir,
+    )
+    all_marathon_tasks, all_mesos_tasks = get_all_marathon_mesos_tasks(marathon_client)
+    if service_config.get_max_instances() and service_config.get_desired_state() == 'start':
+        marathon_tasks, mesos_tasks = filter_autoscaling_tasks(marathon_client,
+                                                               all_marathon_tasks,
+                                                               all_mesos_tasks,
+                                                               service_config)
+
+        autoscaling_params = service_config.get_autoscaling_params()
+        autoscaling_params.update({'noop': True})
```

Comment:
It doesn't does it? https://github.com/Yelp/paasta/blob/93d140eac5fcde50eb0b892494dc7031c7408c4a/paasta_tools/marathon_tools.py#L202-L208

Just fetching the config as far as I can see?